docs/tutorials/data_manipulation/dataset_splitting_example.md.template

```diff
@@ -0,0 +1,118 @@
+# Splitting datasets into training and test
+
+In this notebook, we are going to show that given a dataset how can we split it when necessary. Specifically, we'll use the `split` function that already existed in our GluonTS project.
+
+In the `split` function:
+- you need to provide the `dataset` for the method to split.
+- you also need to provide `offset` or `date`, but not both simultaneously. These two arguments are provided for the method to know how to slices training and test data based on a fixed integer offset or a ``pandas.Period``.
+As a result, the `split` method returns the splited dataset, consisting of the training data `training_dataset` and the TestTemplate objectives `test_template` which knows how to generate test data `test_pairs` using the memeber function `generate_instances`.
+
+
+## Data loading and processing
+
+
+```python
+from gluonts.dataset.field_names import FieldName
+from gluonts.dataset.pandas import PandasDataset
+from gluonts.dataset.split.splitter import split
+from gluonts.dataset.util import to_pandas
```

Comment:
I would put the first import for `split` on top of the notebook, when we first mention `split`.
The import for `to_pandas` can go below when it's used, that is when you introduce the plotting helpers.